src/board/dom-helpers.mjs

```diff
@@ -0,0 +1,15 @@
+function removeElementByClass(className) {
+  // removes the element if it already exists
+  const currentElement = document.querySelector(className);
+  if (currentElement !== null) {
```

Comment:
i think you could probably just check for if it exists here rather than if it is not null

`if (currentElement) currentElement.remove()`

Reply:
✔️